dvc/repo/plots/__init__.py

```diff
@@ -87,7 +87,23 @@ def collect(
                             data[rev][repo_path]["data"] = fd.read()
                     except FileNotFoundError:
                         # This might happen simply because cache is absent
-                        pass
+                        logger.warning(
+                            (
+                                "Could not find '%s' on '%s'. "
+                                "File will not be plotted"
+                            ),
+                            path,
+                            rev,
+                        )
+                    except UnicodeDecodeError:
+                        logger.warning(
+                            (
+                                "'%s' at '%s' is binary file. It will not be "
+                                "plotted."
+                            ),
+                            path,
+                            rev,
+                        )
```

Comment:
We don't use warnings in metrics and params, right? If so, it is better to use `debug()` here too, for consistency.